### PR TITLE
Fix AUTH header issue force en_US locale

### PR DESCRIPTION
**Pull request by fermulator**
Fix AUTH header issue for users outside of en_US locale
As per https://github.com/stianaske/pybotvac/issues/30

### CODE REVIEW

File: pybotvac/robot.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import requests
 import hashlib
```
Comment (fermulator) [inline pybotvac/robot.py:1]: also re-ordered import statements alphabetically

File: pybotvac/robot.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 import hmac
-import time
+import locale
```
Comment (fermulator) [inline pybotvac/robot.py:3]: added locale

File: pybotvac/robot.py [hunk 3]
```diff
@@ -234,6 +235,10 @@ def __init__(self, serial, secret):
         self.secret = secret
 
     def __call__(self, request):
+        # Due to https://github.com/stianaske/pybotvac/issues/30
+        # Neato expects and supports authentication header ONLY for en_US
+        locale.setlocale(locale.LC_ALL, 'en_US.utf8')
```
Comment (fermulator) [inline pybotvac/robot.py:240]: this is the real fix;  obviously forcing locale is unfortunate ... but if the vendor (Neato) doesn't support any other locale other than `en_US`, I don't think there is anything else we can do

(perhaps the RIGHT thing would be to submit a bug report to them)

Comment (fermulator) [reply pybotvac/robot.py:240]: one consideration is that we might want to UNSET (restore) the locale, let me know if that is desired

Comment (Santobert) [reply pybotvac/robot.py]: Hey @fermulator. Thank you for this fix. Unfortunately this causes errors on systems, where en_US.UTF-8 is not available. Can you use pytz instead? https://pypi.org/project/pytz/

**Pull request merged**